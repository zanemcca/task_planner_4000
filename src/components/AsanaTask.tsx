
import React from 'react';
import {
  Row,
  Card,
} from 'antd'

import '../styles/Task.css';
import { ITask } from './Task';

export type IAsanaTaskProps = Pick<ITask, 'description'>

const AsanaTask = (props: IAsanaTaskProps) => {
  return (
    <Card className="Task">
      <Row className="Task-header">
        <div className="Task-description">{props.description}</div>
      </Row>
    </Card>
  )
}

export default AsanaTask;
