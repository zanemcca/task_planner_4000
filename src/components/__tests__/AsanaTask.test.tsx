import renderer from 'react-test-renderer';

import AsanaTask, { IAsanaTaskProps } from '../AsanaTask';
import moment from 'moment';
import React from 'react';

it('should render in a nice way', () => {
  const props: IAsanaTaskProps = {
    description: 'Aliquam vel massa accumsan, accumsan ex eu, posuere nunc.',
  }
  const component = renderer.create( <AsanaTask {...props}/>)
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
