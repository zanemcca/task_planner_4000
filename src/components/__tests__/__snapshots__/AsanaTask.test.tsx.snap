// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render in a nice way 1`] = `
<div
  className="ant-card Task ant-card-bordered"
>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="ant-row Task-header"
      style={Object {}}
    >
      <div
        className="Task-description"
      >
        Aliquam vel massa accumsan, accumsan ex eu, posuere nunc.
      </div>
    </div>
  </div>
</div>
`;
